### PR TITLE
Update window function

### PR DESCRIPTION
**Pull request by kendallm360**
Worked on the function to change the view when the button is clicked. Please review

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -27,29 +27,61 @@ var currentCover = Cover
...
 
-randomCoverButton.addEventListener("click", changeButton);
-function changeButton() {
-currentTitle.innerText = getRandomBookInfo(titles);
-//currentImageSrc.innerText = getRandomBookInfo(covers);
-currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
+//redo for hide element
+function hideElement(element) {
+  document.querySelector(`.${element}`).classList.add("hidden");
+}
```
Comment (jordy1611) [inline src/main.js]: These helper functions will help clean up code.

File: src/main.js [hunk 2]
```diff
@@ -27,29 +27,61 @@ var currentCover = Cover
...
 
-currentTagLine1.innerText = getRandomBookInfo(descriptors);
-currentTagLine2.innerText = getRandomBookInfo(descriptors);
+
+function clickHandler(event) {
+  if (event.target.classList.contains("make-new-button")){
+    showCustomCover()
+  } else if (event.target.classList.contains("random-cover-button")){
+    changeButton()
+  }
```
Comment (jordy1611) [inline src/main.js]: Play around with this logic to see if you can eliminate mosts global event listeners and query selectors.

File: src/main.js [hunk 3]
```diff
@@ -27,29 +27,61 @@ var currentCover = Cover
...
 
 var randomCoverButton = document.querySelector(".random-cover-button");
+var hideHomeView = document.querySelector(".home-view");
+var showFormView = document.querySelector(".form-view");
 var currentTitle = document.querySelector('.cover-title');
-var currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
+var currentImage = document.querySelector('.cover-image');
 var currentTagLine1 = document.querySelector('.tagline-1');
 var currentTagLine2 = document.querySelector('.tagline-2');
+var makeCoverButton = document.querySelector(".make-new-button");
```
Comment (jordy1611) [inline src/main.js]: Naming is a bet confusing. Only use verbs for functions/methods

**Review by jordy1611 — APPROVED** [review]
Good initial work. Using helper functions and global event listeners will help keep code clean and organized.

**Review by jordy1611 — APPROVED** [review]
Good initial work. Using helper functions and global event listeners will help keep code clean and organized.

File: src/main.js [hunk 4]
```diff
@@ -13,43 +13,83 @@ var currentCover = Cover
...
-
 
+var homeView = document.querySelector(".home-view");
+var showFormView = document.querySelector(".form-view");
+var savedView= document.querySelector(".saved-view");
 var randomCoverButton = document.querySelector(".random-cover-button");
+var homeButton = document.querySelector(".home-button");
+var saveCoverButton = document.querySelector(".save-cover-button");
 var currentTitle = document.querySelector('.cover-title');
-var currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
```
Comment (shanekwarning) [inline src/main.js:31]: Great removal of overly cumbersome code.

File: src/main.js [hunk 5]
```diff
@@ -13,43 +13,83 @@ var currentCover = Cover
...
+  homeButton.classList.remove("hidden");
+}
+
+function viewSavedCovers() {
+  homeView.classList.add("hidden");
+  showFormView.classList.add("hidden");
+  savedView.classList.remove("hidden");
+  randomCoverButton.classList.add("hidden");
+  saveCoverButton.classList.add("hidden");
+  homeButton.classList.remove("hidden");
```
Comment (shanekwarning) [inline src/main.js:70]: These are great functions and work. Since there are many that are similar may want to use a class in the future.

**Review by shanekwarning — APPROVED** [review]
Overall a great update. This makes the buttons and pages on the files appear and function properly on the GUI.

**Pull request merged**